mvp-spec/dsp-implementation.md

```diff
@@ -42,8 +42,8 @@ we can trust that it comes from the actual **Sender**. It is associated with
 a **Transaction** via a **Seed**.
 
 A **Seed** is data generated by the publisher or one of its direct third parties. 
-It gathers the identity of the publisher and a **Transaction Id**. The Seed is signed
-by the actor that generated it so that it can be verified.
+It gathers the identity of the publisher, a **Transaction Id** and a list of **Addressable Content Id**. 
```

Comment:
I'm wondering whether we should instead just have a list of transaction id (1 per addressable content).

If I remember correctly, this would align better to what regulator expect a "transaction" to be - or am I mistaken ?

Reply:
After our informal discussion, I revamp for having one Seed for many Addressable Content and one Transaction-Id per Addressable Content.